client/state/sites/vouchers/actions.js

```diff
@@ -103,3 +113,33 @@ export function requestSiteVouchers( siteId ) {
 			} );
 	};
 }
+
+/**
+ * Assign a voucher to the given site.
+ *
+ * @param {Number} siteId - identifier of the site
+ * @param {String} serviceType - service type
+ * @returns {Function} a promise that will resolve once fetching is completed
+ */
+export function assignSiteVoucher( siteId, serviceType ) {
+	return dispatch => {
+		dispatch( vouchersRequestAction( siteId ) );
+
+		return wpcom
+			.site( siteId )
+			.adCreditVouchers()
+			.assign( serviceType )
```

Comment:
okay, I think I read it wrong, though I still couldn't find these functions defined inside of Calypso. Am I missing something major?